### PR TITLE
fix chat

### PR DESCRIPTION
**Pull request by tf-4**
... verb

find a proper fix for this soon:tm:

:cl: Gandalf2k15
admin: Added verb to fix chat called "FIX SAY".
/:cl:

### CODE REVIEW

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Forgot to tick the code file

**Comment by tf-4** [comment]
Right as I get in bed, too

File: modular_skyrat/modules/admin/code/fix_chat.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/client/proc/fix_say()
+    set name = "FIX SAY"
+    set category = "Admin"
+    set desc = "fixes bug where people can't say shid"
+    for(var/player in GLOB.player_list)
```
Comment (Iamgoofball) [inline modular_skyrat/modules/admin/code/fix_chat.dm:5]: this list is supposed to be cleaning itself of nulls, i suspect you have a runtime causing this problem

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -96,6 +96,7 @@ GLOBAL_LIST_INIT(admin_verbs_fun, list(
 	/client/proc/one_click_antag, // SKYRAT EDIT ADDITION - ONE CLICK ANTAG
 	/client/proc/spawn_mob_spawner, // SKYRAT EDIT ADDITION
 	/client/proc/request_more_opfor, //SKYRAT EDIT ADDITION
+	/client/proc/fix_say, // SKYRAT EDIT ADDITION
```
Comment (Iamgoofball) [inline code/modules/admin/admin_verbs.dm:99]: put it in the .dme

Comment (tf-4) [reply code/modules/admin/admin_verbs.dm:99]: I know, I'll fix it tomorrow

File: modular_skyrat/modules/admin/code/fix_chat.dm (hunk 1 above)
Comment (tf-4) [reply modular_skyrat/modules/admin/code/fix_chat.dm:5]: Apparently it happens on tg and bee as well

**Review by Gandalf2k15 — APPROVED** [review]

File: modular_skyrat/modules/admin/code/fix_chat.dm (hunk 1 above)
Comment (Kapu1178) [reply modular_skyrat/modules/admin/code/fix_chat.dm:5]: This isnt a problem on Beestation or Shiptest

**Pull request merged**